Makefile

```diff
@@ -0,0 +1,30 @@
+SOURCE_FILES?=$$(glide novendor)
+TEST_PATTERN?=.
+TEST_OPTIONS?=
+
+setup: ## Install all the build and lint dependencies
+	curl https://glide.sh/get | sh
+	go get -u github.com/kisielk/errcheck
+	go get -u github.com/golang/lint/golint
+	glide install
+
+test: ## Run all the tests
+	go test $(TEST_OPTIONS) -cover $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=30s
+
+lint: ## Run all the linters
+	errcheck $(SOURCE_FILES)
+	go vet $(SOURCE_FILES)
+	golint ./... | grep -v vendor
+
+ci: lint test ## Run all the tests and code checks
+
+build: ## Build a beta version of releaser
+	go build
```

Comment:
the custom build id right now is the tag name. You can see it [here](https://github.com/goreleaser/releaser/blob/master/main.go#L16) and [here](https://github.com/goreleaser/releaser/blob/master/pipeline/build/build.go#L43).